### DISCUSSION TITLE
PA Bias - non linear/weird power

### DISCUSSION THREAD

**Discussion by danielebruneo** [df8oe/UHSDR#1944] [General]
Hi all,
I'm experiencing some strange behavior trying to properly set PA Bias and PWR adj for the first time.

When setting PA Bias, at some point power just jumps from 450~500mA to over 1A.
If I succeed to set PA Bias to ~500mA (i.e. ~940mA), ignoring the fact that RF is being produced, and maybe also able to set the 5W PWR for let's say 10m... then 2W produce like 4W, 1W does 3.5W, 0.5W does 8W or more....

There's definitely something wrong... either PA oscillating on multiple points, that slightly vary on supply voltage... or I don't know maybe a blown final.

Setting PA Bias there are at least a couple of weird points:
- I get it slightly below 80, no RF power (which is what we want if I get it right), too small mA bias though... 
- increase it at 80-85 or slightly above, RF power starts going crazy (i.e.7-8W)
- then I have another point at ~105 in which RF power goes down to zero again, but with a bias of ~7-800mA
Also... when I set 5W power gain: I have a point (like below 25 if I remember right) in which RF power being a lot more than 5W (i.e.8W), then it drops and gets linear for a bit but can't reach 5W though (i.e. 4ish)
Then.... 0,5W is still behaving like full power.

Additionally, I checked the PA_BIas output from DAC1 at Pin8 and it increases linearly... the threeshold where weirdness happens is around roughly 2.53V at PA_Bias.
So, it's not a DAC issue... should be downhill from that.

btw, this is a pre-assembled chinese rig (this basically being an MCHF 0.6 with the Si5351a oscillator)... so entirely possible components (i.e. finals) are not of the highest quality... though it was apparently behaving fine at the beginning. Wonder if I may have burnt something during the calibration operation in which some super-high SWR came out (probably due to PA oscillating and producing noise)...

I also added a 10uF 16V tantalum capacitor in parallel to C106 but nothing changed.

Any ideas?

**Comment by df8oe** [comment]
I think it is oscillation. Its definitely not a firmware issue. You have good chances to get answers at our discussion group https://www.amateurfunk-sulingen.de/forum/index.php . I do not want to scatter the discussions over multiple places...

**Comment by danielebruneo** [comment]
I can confirm that replacing the finals (sill using RD16HF) solved the issue and I'm now able to get consistent power output.

So I suspect it was oscillation due to either damaged or non genuine mosfets.
